experimental/packages/opentelemetry-instrumentation-http/src/utils.ts

```diff
@@ -91,15 +95,15 @@ import forwardedParse = require('forwarded-parse');
 export const getAbsoluteUrl = (
   requestUrl: ParsedRequestOptions | null,
   headers: IncomingHttpHeaders | OutgoingHttpHeaders,
-  fallbackProtocol = 'http:'
+  fallbackProtocol = 'http:',
+  redactedQueryParams?: string[]
```

Comment:
This should allow us to avoid some of the if-else branching statement below I think

```suggestion
  redactedQueryParams?: string[] = DEFAULT_QUERY_STRINGS_TO_REDACT
```